src/thinkspace/api/thinkspace-readiness-assurance/db/associations.yml

```diff
@@ -4,7 +4,7 @@
     - {belongs_to: thinkspace/common/user, readonly: true}
     - {has_many: thinkspace/readiness_assurance/responses, dependent: destroy}
   serializers:
-    - attributes: [id, title, authable_type, authable_id, question_settings, ra_type]
+    - attributes: [id, title, authable_type, authable_id, question_settings, questions, settings, ra_type, answers]
```

Comment:
This can not serialize `answers`, it needs to be removed.  Is it required for your functionality?  If we leave this in, it would leak the answers to the client.  The `answers` are added automatically to updaters on the assignment by  get added in automatically when the user can update the record - `cellar/src/thinkspace/api/thinkspace-readiness-assurance/app/controllers/thinkspace/readiness_assurance/api/admin/irats_controller.rb`